Formula/d/dnsviz.rb

```diff
@@ -31,21 +31,6 @@ class Dnsviz < Formula
     depends_on "swig"
   end
 
-  resource "dnspython" do
-    url "https://files.pythonhosted.org/packages/65/2d/372a20e52a87b2ba0160997575809806111a72e18aa92738daccceb8d2b9/dnspython-2.4.2.tar.gz"
-    sha256 "8dcfae8c7460a2f84b4072e26f1c9f4101ca20c071649cb7c34e8b6a93d58984"
-  end
-
-  resource "M2Crypto" do
-    url "https://files.pythonhosted.org/packages/16/49/bfeaf55da3378292e40e93319717bb4334400e86c00a57fc52677f11fb65/M2Crypto-0.39.0.tar.gz"
-    sha256 "24c0f471358b8b19ad4c8aa9da12e868030b65c1fdb3279d006df60c9501338a"
-  end
-
-  resource "pygraphviz" do
-    url "https://files.pythonhosted.org/packages/19/db/cc09516573e79a35ac73f437bdcf27893939923d1d06b439897ffc7f3217/pygraphviz-1.11.zip"
-    sha256 "a97eb5ced266f45053ebb1f2c6c6d29091690503e3a5c14be7f908b37b06f2d4"
-  end
-
```

Comment:
$ pipgrip --tree dnsviz
dnsviz (0.9.4)

Reply:
Looks like the dependencies aren't actually listed in the metadata used by pip, they're in a seperate requirements.txt https://github.com/dnsviz/dnsviz/blob/master/requirements.txt